tic_tac_toe/game.py

```diff
@@ -47,7 +55,7 @@ def _is_winning_combination(board, combination, player):
     Returns True of all three positions in the combination belongs to given
     player, False otherwise.
     """
-    pass
+    return not False in [board[i][j] == player for i,j in combination]
```

Comment:
Does `not False` == `True`? If so maybe don't use a double negative here for readability.